charts/airbyte/templates/secret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "common.names.fullname" . }}-secrets
+  annotations:
+    helm.sh/hook: pre-install,pre-upgrade
+    helm.sh/hook-weight: "-1"
+type: Opaque
+stringData:
+  AWS_ACCESS_KEY_ID: {{ .Values.logs.accessKey.password | quote }}
+  AWS_SECRET_ACCESS_KEY: {{ .Values.logs.secretKey.password | quote }}
+  DATABASE_PASSWORD: {{ .Values.postgresql.postgresqlPassword }}
```

Comment:
I'm wondering if this key is useful:
If you use an external postgres the secret is stored in `external-db.yaml`. If you use the postgres chart the secret is stored in `.Values.postgresql.existingSecret` or in `airbyte.postgresql.fullname`.

The `_helpers.tpl` is responsible of mapping the correct secret to `airbyte.postgresql.secretName` and `airbyte.database.existingsecret.key`.

Reply:
i can probably move this. At the time that I raised this pr, external-db.yaml didn't exist but it appeared after a rebase. But to be honest, it would probably be better to remove the external-db.yaml secret completely and use just the secret that I created in this PR. I can rework it that way. What do you think?

Reply:
Agreed!